### DISCUSSION TITLE
How to hide dock icons

### DISCUSSION THREAD

**Discussion by mjdNjhNJ** [zed-industries/zed#15811] [Feature Requests]
I would like to hide dock icons like **Assistant Panel**, **Chat Panel**, **Notification Panel** from the dock. Is there a possibility?

<img width="503" alt="image" src="https://github.com/user-attachments/assets/92a57f54-9357-4bec-be7f-e01261fcba61">

**Comment by bajrangCoder** [comment]
Use these settings to hide those mentioned icons:

Assistant Panel:
```json
"assistant": {
  "button": false
}
```

Chat panel:
```json
"chat_panel": {
    "button": false,
},
```

I'm not sure about Notification icon

**Reply by mjdNjhNJ** [reply]
Thanks. Do you also know how to hide **Collab panel**?

**Reply by bajrangCoder** [reply]
```json
"collaboration_panel": {
    "button": false,
},
```

**Comment by mjdNjhNJ** [comment]
Here are the settings that hides **Assistant Panel**, **Chat Panel** and **Notifications Panel** from the dock.

```
// Hide specific panels from the dock
"assistant": {
	"version": "2",
        "button": false
},
"chat_panel": {
	"button": false
},
"notification_panel": {
	"button": false
},
"collaboration_panel": {
    "button": false,
},
```
